crates/ark/src/variables/variable.rs

```diff
@@ -1779,6 +1818,14 @@ mod tests {
         })
     }
 
+    fn inspect_from_expr(code: &str) -> Vec<Variable> {
+        let env = harp::parse_eval_global("new.env()").unwrap();
```

Comment:
Probably worth adding an `Environment` method at some point.

Reply:
Added in https://github.com/posit-dev/ark/pull/629/commits/840a877501e581ea0d3b93ae683b3e0513437dd9, let me know what you think?

Reply:
Seems reasonable